x-pack/apm-server/fields/_meta/fields.yml

```diff
@@ -17,3 +17,20 @@
           type: histogram
           description: >
             Pre-aggregated histogram of transaction durations.
+
+- key: apm-span-metrics-xpack
+  title: "APM Span Metrics"
+  description: >
+    APM span metrics are used for showing rate of requests and latency between instrumented services.
+  short_config: true
+  fields:
+    - name: destination.service
+      type: group
+      dynamic: false
+      fields:
+        - name: respose_time.count
+          type: long
+        - name: response_time.count_interval
+          type: long
```

Comment:
This field name is too specific - the aggregation interval covers both duration and count.

How about we call this `metricset.period` to align with Metricbeat, so we have something reusable? We may want to use this for other metrics too: transaction metrics, and also the metrics published by agents. Metrics Explorer already knows about the `metricset.period` field, and I'm thinking that by using the same field name we would be setting up the APM app to be able to embed Metrics Explorer for visualising APM metrics.